azure-configurationservice/tests/conftest.py

```diff
@@ -0,0 +1,113 @@
+import os
+import logging
+from msrestazure.azure_exceptions import CloudError
+from azure.configuration import AzureConfigurationClient
+from azure.configuration import ConfigurationSetting
+
+PAGE_SIZE = 100
+
+
+class AzConfigTestData:
+    def __init__(self):
+        self.key_uuid = None
+        self.label1 = None
+        self.label2 = None  # contains reserved chars *,\
+        self.label_uuid = None
+        self.label1_data = []
+        self.label2_data = []
+        self.no_label_data = []
+
+
+def _add_for_test(app_config_client, kv):
+    exist = bool(list(app_config_client.list_configuration_settings(keys=[kv.key], labels=[kv.label])))
```

Comment:
If you are querying for a specific key-label , why not use Get, instead of List?

Reply:
both get and list will work. I use list just for code simplicity. As you can see, one statement is enough. If I use get, I'll need to try ... except ...